File: PYTHON-SQL PROBLEM SETS/problem set 1 python/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height < 1 or height > 8:
        height = get_int("Height: ")
    break

for i in range(1, height + 1):
    print(" " * (height - i) + "#" * i)

File: C-PROBLEM SETS/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int altura;
    int count = 0;
    do
    {
        altura = get_int("Altura del bloque: ");

    }
    while (altura < 1 || altura > 8);

    for (int i = 0; i < altura; i++)
    {
        for (int j = altura - 1; j > i; j--)
            printf(" ");
        for (int j = 0; j <= i; j++)
            printf("#");
        printf("\n");

            }


    }


